src/utilities/json.ts

```diff
@@ -23,3 +25,12 @@ export function stringifyUnknownJson(value: unknown): string | null | undefined
 
   return stringify(parsed)
 }
+
+export function isValidJsonString(value: unknown): value is string {
+  try {
+    const parsed = JSON.parse(value as string)
+    return isRecord(parsed)
+  } catch {
+    return false
+  }
+}
```

Comment:
I think this method is misleading. `'true'` is a valid json string but if passed to this method it would return false. I think `isValidJsonRecord` would be more accurate.